base/system/winlogon/sas.c

```diff
@@ -4,7 +4,7 @@
  * FILE:            base/system/winlogon/sas.c
  * PURPOSE:         Secure Attention Sequence
  * PROGRAMMERS:     Thomas Weidenmueller (w3seek@users.sourceforge.net)
- *                  Herv� Poussineau (hpoussin@reactos.org)
+ *                  Hervé Poussineau (hpoussin@reactos.org)
```

Comment:
Encoding change, please fix.

Reply:
What is the encoding I should use for this file?

Reply:
Whatever it was